components/list/list.js

```diff
@@ -1,17 +1,14 @@
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { SelectionMixin, selectionStates } from '../selection/selection-mixin.js';
 
-export const listSelectionStates = {
-	none: 'none',
-	some: 'some',
-	all: 'all'
-};
```

Comment:
Good question.  Yeah I am not sure if we will need it or not.  Related to that is the question of what "select all" means - that is, is it all on the page, or all for all pages?  I think `all-except` is only really useful if select-all applies to _all_ all. For that scenario with paging, consumers need to figure out what to do on the backend since the client doesn't have all the keys.
 
I think this probably needs some more discussion to figure out what we want design-wise, and what the new APIs are supporting. IIRC, the MVC controls provide a way to select all, and also _all_ all.